build/config.props

```diff
@@ -11,7 +11,7 @@
   <!-- Version -->
   <PropertyGroup>
     <!-- **  Change for each new version -->
-    <!-- when changing any of the NuGetVersion props below, also update ProductVersion in src\NuGet.Clients\NuGet.Tools\NuGetPackage.cs -->
+    <!-- when changing any of the NuGetVersion props below, update ProductVersion in src\NuGet.Clients\NuGet.Tools\NuGetPackage.cs and run tools-local\ship-public-apis -->
```

Comment:
I think this is relevant and important enough that I think should be a part of the client.engineering issue template. 

what do you think?

Reply:
Yes, that was my intention. I didn't want to create a PR on the template before this gets merged though :)

Reply:
makes sense. Just make sure you follow up on it. 
you can resolve this thread.